### ISSUE TITLE
Update docs to have link to donation link.  :)

### ISSUE THREAD

**Issue by NathanaelA** [hirschmann/nbfc#701]
**Is your feature request related to a problem? 
Not easy to find link...

**Describe the solution you'd like**
A link in the docs for donation.  It is in the windows code; but most people aren't going to scan the windows code to find it, if they are using linux...  :D

**Additional context**
https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=HUALCC9HY9MKC

**Comment by hirschmann** [comment]
Fixed :D
I hope a paypal link in the readme isn't considered too annoying.

**Comment by NathanaelA** [comment]
@hirschmann - I don't think anyone finds a paypal link in the docs annoying.   But it will help those of us on Linux (which I run 99% of the time) be able to donate a lot lot easier...   

I wouldn't have had a clue where to donate to, if I hadn't noticed earlier in the day a link in the Windows version that I had also just installed.   So, I knew that I could probably find it somewhere in the source code... :-)     

I am VERY happy this app supports Linux -- I had tried a dozen or so apps/ways several months back and none of them worked, so I had basically given up.     And so my computer frequently overheats and throttles down my cpus to crazy slow speeds...   So solving a major headache for me, was well worth a donation...  :grinning:
